src/index.ts

```diff
@@ -224,39 +205,25 @@ export default <ExportedHandler<Env>>{
 								}
 							),
 						];
-						if (answerState.CmdName === NeedsMoreInfo.CmdName && !message.thread) {
+						if (answerState.CmdName === NeedsMoreInfo.CmdName) {
```

Comment:
Just to be safe - it's possible the thread was deleted for whatever reason
```suggestion
						if (answerState.CmdName === NeedsMoreInfo.CmdName && message.thread) {
```


We could also potentially check if `answerState.CmdName !== message.embeds[0].title`? If it's already equal - the bot is likely to have already sent a message asking for this. (Could happen when 2 people click needs more info at the same time). This is very much a nitpick though - not needed to merge